DZ_4/potoc.py

```diff
@@ -0,0 +1,44 @@
+import requests
+import os
+import threading
+import time
+import sys
+
+
+def download_image(url):
+    try:
+        response = requests.get(url)
+        if response.status_code == 200:
+            # filename = "DZ_4/foto/" + url.split('/')[-1]
+            filename = "foto/" + url.split('/')[-1]
```

Comment:
То есть можно будет сделать вот так
```suggestion
            filename = os.path.join(folder, os.path.basename(url))
```
Такой код будет работать вне зависимости от платформы.